src/SourcemapToolkit.SourcemapParser/Base64VlqEncoder.cs

```diff
@@ -20,12 +20,14 @@
 
 namespace SourcemapToolkit.SourcemapParser
 {
+	using System.Text;
```

Comment:
Can you move this to be next to the other using statement in this file instead of inside the namespace?

Reply:
+